html/modules/custom/reliefweb_import/src/Command/ReliefwebImportCommand.php

```diff
@@ -576,6 +576,27 @@ protected function validateHowToApply($data) {
     return $field_how_to_apply;
   }
 
+  /**
+   * Validate city field.
+   *
+   * @param string $data
+   *   Raw data from XML.
+   */
+  protected function validateCity($data) {
+    // Clean the field.
+    $field_city = $this->sanitizeText('field_city', $data, 'markdown');
```

Comment:
@attiks The city field only accepts a plain text string. Sanitation should probably do something like that instead to be closer to the D7 validation:

```
$field_city = TextHelper::cleanText(strip_tags($data), [
  'line_breaks' => TRUE, 
  'consecutive' => TRUE,
]);
```

Reply:
Changed, thanks